src/MailTracking.php

```diff
@@ -90,6 +87,38 @@ public function recordMail(Swift_Message $email)
     }
 
     /**
+     * Assert that the last email was bcc'ed to the given address.
+     *
+     * @param string        $bcc
+     * @param Swift_Message $message
```

Comment:
Do you want to use that, or the convention `@param Swift_Message|null $message`?